packages/frontend/web/components/Outbrain.tsx

```diff
@@ -3,14 +3,47 @@
 import React from 'react';
 import { Container } from '@guardian/guui';
 
-export const shouldDisplayOutbrain = (config: ConfigType): boolean => {
+interface OutbrainSelectors {
+    widget: string;
+    container: string;
+}
+
+const shouldDisplayOutbrain = (config: ConfigType): boolean => {
     return config.switches.outbrainDCRTest;
 };
 
-export const OutbrainWidget: React.FC<{}> = ({}) => {
+type OutbrainSelectorsType = 'outbrain' | 'merchandising' | 'nonCompliant';
+
+const outbrainSelectorsTypeMapping = {
+    outbrain: {
+        widget: 'js-outbrain',
+        container: 'js-outbrain-container',
+    },
+    merchandising: {
+        widget: 'js-container--commercial',
+        container: 'js-outbrain-container',
+    },
+    nonCompliant: {
+        widget: 'js-outbrain',
+        container: 'js-outbrain-container',
```

Comment:
I wrote a bunch about how this was overly complex (same as Phil I guess) because the containers are always the same and the widgets only have two variations, but then it's difficult to review as the existing code does a lot more and obviously this is cut down. 

I guess it makes sense to refactor this when it's more complete but really it's more of this code, as is, looks fine that anything useful from a review.

Maybe the 'most simple' would make more sense, introducing complexity with functionality, rather than recreating the complexity piecemeal without the functionality?

Reply:
Yep +1 for refactoring this one day...